bundler/lib/bundler/definition.rb

```diff
@@ -671,14 +674,18 @@ def add_current_platform
 
     def change_reason
       if unlocking?
-        unlock_reason = @unlock.reject {|_k, v| Array(v).empty? }.map do |k, v|
-          if v == true
-            k.to_s
-          else
-            v = Array(v)
-            "#{k}: (#{v.join(", ")})"
-          end
-        end.join(", ")
+        stuff_to_unlock = if @gems_to_unlock.any?
```

Comment:
_:smiley_cat:  on the naming, but I have no better idea :fearful:_

Reply:
Haha, yeah, maybe `unlock_targets`?

Reply:
I went with `unlock_targets` since it seemed a bit more "formal" 😅

Reply:
sounds good